gnssanalysis/gn_io/igslog.py

```diff
@@ -109,21 +134,25 @@ def parse_igs_log(filename_array: _np.ndarray) -> _np.ndarray:
     with open(file_path, "rb") as file:
         data = file.read()
 
-    blk_id = _REGEX_ID.search(data)
+    blk_id = _REGEX_ID_V2.search(data)
     if blk_id is None:
-        tqdm.write(f"ID rejected from {file_path}")
-        return _np.array([]).reshape(0, 12)
+        blk_id = _REGEX_ID.search(data)
+        if blk_id is None:
+            tqdm.write(f"ID rejected from {file_path}")
```

Comment:
There is just 2 files in the library that uses tqdm (top.py too), shall we think about removing it?

Reply:
Done